data/gigadb_testdata/dataset_author.csv

```diff
@@ -19,3 +19,14 @@ id,dataset_id,author_id,rank,role
 3490,212,3807,5,
 3491,213,3808,1,
 3492,213,3809,2,
+14014,2335,14014,1,
+14015,2335,14015,2,
+14019,2335,14019,3,
+14020,2335,14020,4,
+14016,2335,14016,5,
+14017,2335,14017,6,
+14021,2335,14021,7,
+14022,2335,14022,8,
+14023,2335,14023,9,
+14013,2335,14013,10,
+14018,2335,14018,11,
```

Comment:
I would list the new records by order of `id` so they are in sequence with existing records.